src/lib/accessibility/FocusZone/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 This is a list of changes made to this Stardust copy of FocusZone in comparison with the original [Fabric FocusZone @ 0f567e05952c6b50c691df2fb72d100b5e525d9e](https://github.com/OfficeDev/office-ui-fabric-react/blob/0f567e05952c6b50c691df2fb72d100b5e525d9e/packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx).
 
+### fix(FocusZone)with defaultTabbableElement prop set tabindexes are not updated accordingly [#342](https://github.com/stardust-ui/react/pull/342)
```

Comment:
Please also add similar entry to public changelog.

Reply:
will do, thanks